index.html

```diff
@@ -55,88 +61,196 @@ <h6>LET'S CONNECT</h6>
+              <h3>Uber Navigation</h3>
+              <ul class="flex project_details">
+                <li>Uber</li>
+                <li>Lead Developer</li>
+                <li>2018</li>
+              </ul>
+              <p>
+                A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive
+                computer: your car.
+              </p>
+              <ul class="flex project_coding_langs">
+                <li>html</li>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+              <button class="preview_button outlinings">See Project</button>
+            </div>
+          </article>
+
+
+          <article class="hide card flex white-back">
+            <div class="desktop-wrok-img">
+              <img src="images/desktop-Snapshoot3.svg" alt="tonic project preview" />
+            </div>
+            <img class="mobile-work-img" src="./images/project_3.png" alt="tonic project preview" />
+            <div class="desktop-work-detail">
+              <h3>Tonic</h3>
+              <ul class="flex project_details">
+                <li>CANOPY</li>
+                <li>BACKEND DEV</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="flex project_coding_langs">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+              <button class="preview_button outlinings">See Project</button>
+            </div>
+          </article>
+          <article class="hide reverse-work-2 card flex white-back">
+            <div class="desktop-wrok-img">
+              <img src="images/desktop-Snapshoot4.svg" alt="tonic project preview" />
+            </div>
+            <img class="mobile-work-img" src="./images/project_4.png" alt="tonic project preview" />
+            <div class="desktop-work-detail">
+              <h3>Multi-Post Stories</h3>
+              <ul class="flex project_details">
+                <li>CANOPY</li>
+                <li>BACKEND DEV</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="flex project_coding_langs">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+              <button class="preview_button outlinings">See Project</button>
+            </div>
+          </article>
+        </div>
       </section>
+
       <section class="about white-back">
-        <div>
+        <div class="desktop-about">
           <h1>
             About <br />
             Myself
```

Comment:
- [ ] Great job on designing the desktop version of the design 👏 The headline and the project cards look amazing 💯 However, the `about me` section still needs some work to make it match Figma. the sizes of the sections (left part and right part) are different from Figma as you can see here:

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192902377-891e6f4e-4610-4d19-aab8-7e217d5f6525.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192902395-4f705eb2-e75c-4f30-894f-0670405c1ba5.png)

- [ ] The languages list is different than the one in Figma. In Figma, the text and the icons are a column instead of a row and there is some space between the languages 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192902558-0c75c2de-b07a-477b-809b-d0f1a658b028.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192902579-d6c8bab1-bcf0-4703-bddd-ec4e034ae1b0.png)